src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,107 @@
+package duke.task;

```

Comment:
My comments on Deadline.java apply here too!

Reply:
Thanks for your input!